lib/system/widestrs.nim

```diff
@@ -144,11 +147,13 @@ proc newWideCString*(source: cstring, L: int): WideCStringObj =
   result[d] = Utf16Char(0)
 
 proc newWideCString*(s: cstring): WideCStringObj =
+  ## Returns a copy of the string `s` in wide form. `s` should be NUL-terminated.
```

Comment:
```suggestion
  ## Returns a copy of the string `s` in wide form. `s` should be NULL-
  ## terminated.
```

null, as in null byte, is double 'l'; it's a small thing, as the meaning is still clear.

Reply:
`NUL` is the correct spelling for the ASCII null character. Apparently it's an [abbreviation for null](https://en.wikipedia.org/wiki/ASCII#Control_code_chart).

Reply:
LoL, I guess it needed abbreviating.